tokens/contracts/access/MintControl.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+contract MintControl is Initializable, OwnableUpgradeable {
```

Comment:
1. What about `MinterRestriction` for the contract name and functions/events?
`MintControl` is not specific about what control is done.
This contract is about restricting the mint to a specific list of minter, this `MinterRestriction`

2. Second idea: `MinterAccessControl`.
It sounds like `AccessControl` from OpenZeppelin so developer could already assume a lot of functionalities.
`Minter` because it's about adding `Minter` role and not changing the functionality of Mint.

Reply:
I'm good with both suggestions, but `MinterAccessControl` may be more appropriate if the project start to use `AccessControl` later.
I will go with that